tests/init_test.go

```diff
@@ -0,0 +1,51 @@
+package tests
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+	"github.com/shutter-network/shutter-service-api/common"
+	"github.com/shutter-network/shutter-service-api/internal/data"
+	"github.com/shutter-network/shutter-service-api/internal/usecase"
+	"github.com/shutter-network/shutter-service-api/tests/mock"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestShutterService struct {
+	suite.Suite
+	testDB                  *common.TestDatabase
+	dbQuery                 *data.Queries
+	cryptoUsecase           *usecase.CryptoUsecase
+	config                  *common.Config
+	shutterRegistryContract *mock.MockShutterregistry
+}
+
+func TestMain(t *testing.T) {
```

Comment:
getting a wrong signature error as TestMain is reserved for setting up and tearing down tests globally and should be used as follows: 
```
func TestMain(m *testing.M) {
    // Perform setup here (e.g., initialize resources).
    code := m.Run() // Run all tests.
    // Perform teardown here (e.g., release resources).
    os.Exit(code)   // Exit with the proper code.
}
```
test cases should follow this pattern:
```
func TestShutterServiceSuite(t *testing.T) {
    suite.Run(t, new(TestShutterService))
}
```

Reply:
resolved on call